configure.ac

```diff
@@ -79,9 +79,8 @@ AC_CHECK_HEADERS([sys/cdefs.h sys/file.h sys/mman.h sys/param.h \
 AC_CHECK_HEADERS([bzlib.h],
                  [],
                  [AC_MSG_FAILURE([missing <bzlib.h>; is bzip2 installed?])])
-AC_CHECK_HEADERS([json.h],,
```

Comment:
I heard it is good practice when doing `PKG_CHECK_MODULES` to also include the actual check on the lib/headers.

For example, in this [link from Android](https://android.googlesource.com/platform/external/cryptsetup/+/7ae863e380dac362d5df269d9338d83238543b5d/configure.ac) they do the following:

```cmake
dnl Try to detect needed device-mapper static libraries, try pkg-config first.
LIBS="$saved_LIBS -static"
PKG_CHECK_MODULES([DEVMAPPER_STATIC], [devmapper >= 1.02.27],,[
	...
])
LIBS="$saved_LIBS $DEVMAPPER_STATIC_LIBS"
AC_CHECK_LIB(devmapper, dm_task_set_uuid,,
	AC_MSG_ERROR([Cannot link with static device-mapper library.]))
```

And at the end they do:
```cmake
AC_CHECK_HEADER(libdevmapper.h,,[AC_MSG_ERROR([You need the device-mapper library.])])
```